go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -741,22 +779,27 @@ func (h *defaultNetworkControllerEventHandler) AddResource(obj interface{}, from
 
 	case factory.EgressNodeType:
 		node := obj.(*kapi.Node)
+		// Add the node to the tracker only if it belongs to the local zone.
+		klog.Infof("SURYA %v", config.OVNKubernetesFeature.EnableInterconnect)
+		if config.OVNKubernetesFeature.EnableInterconnect {
```

Comment:
we can remove the feature flag because even in default deployments, the `isLocalZone` will mean all nodes are local to the zone.